### DISCUSSION TITLE
production from RP buildings

### DISCUSSION THREAD

**Discussion by Qamadrya** [TownyAdvanced/TownyResources#90] [Q&A] [answered]
Is it possible to make production from RP buildings? Maybe it's not clear what I mean. For example: a player has built a building "Factory", and i use the command to give him 1 lvl townyresources.
By the way, I figured out the commands, it’s just that raw ores are entered in the category, which are not on 1 16 5.

**Comment by LlmDl** [comment, marked as answer]
There is a ticket already on the tracker that will hopefully one day see a survey plot type added to Towny Resources, that is then injected into Towny. This plot type will tie resource collection to a geographical location in a town. The players can then build an RP building on the plot.

**Reply by Qamadrya** [reply]
Does that mean it's not possible at the moment?

**Reply by LlmDl** [reply]
It is not, there is a non-insignificant amount of work that has to be done before that can happen.

**Reply by Qamadrya** [reply]
Is it possible to give /tr survey city by admin? so that the players could not do this, and the admin, as through /townyadmin, issued /tr survey?

**Reply by LlmDl** [reply]
It doesn't look like that is a command, no.
